packages/react/src/metadata.tsx

```diff
@@ -32,11 +32,13 @@ const FieldMetadataFragment = graphql(/* GraphQL */ `
       ... on GadgetHasManyConfig {
         relatedModel {
           apiIdentifier
+          namespace
```

Comment:
what happens if you issue this query against a pre 1.1 app?

Reply:
It returns `[]`